AVR code/MySC1.1.c

```diff
@@ -138,21 +106,31 @@ ISR(USART_RXC_vect)
 			if(UDR!=200)
 			dummy = UDR;
 		}
-		//USART_Transmitchar('202');
+		//USART_Transmitchar('202',0);
 		//ICR1 = 0;
 	}
 	//USART_TransmitNumber(max);
 	//USART_Transmitchar(0x0d);
 }
 
-void main(void)
+int main()
 {
 	sei();
 	pwm_init();
-	USART_Init(12);
-	USART_InterruptEnable();
+	USART_Init(12,0);
+	USART_InterruptEnable(0);
 	OCR1B = 80;
+	
+	TCCR1B |=1<<CS10|1<<WGM12;
```

Comment:
Do not change prescaler of timer which is used to produce sound frequency; either change the timer or adjust OCR1A accordingly. This is timer 1; either use timer 2 or 0 OR change OCR1A according to the time delay you want with CS11 prescaler